### ISSUE TITLE
broken?

### ISSUE THREAD

**Issue by l3iggs** [stuartpb/aur.sh#4]
Has aur4 broken this?

```
$ bash <(curl aur.sh) -si yaourt
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   167  100   167    0     0    342      0 --:--:-- --:--:-- --:--:--   342
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   168  100   168    0     0    888      0 --:--:-- --:--:-- --:--:--   888

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
/dev/fd/63: line 7: cd: yaourt: No such file or directory
==> ERROR: PKGBUILD does not exist.
```

**Comment by l3iggs** [comment]
This PR should fix it (lightly tested): https://github.com/stuartpb/aur.sh/pull/5

**Comment by stuartpb** [comment]
Thanks - this'll probably make it in to https://meta.sh/aur, but I can't say how long it's going to be until it makes it to aur.sh (since that version of the script is currently deployed based on a bunch of hacky Apache rules via FTP).

What would be most helpful is if somebody could push a pr to https://github.com/stuartpb/meta.sh that adds support for CNAME files, but IIRC there's more that has to go into refactoring meta.sh before that can be tenable.

**Comment by l3iggs** [comment]
I've just made a similar PR for meta.sh: https://github.com/stuartpb/meta.sh/pull/6

**Comment by AndrewBelt** [comment]
> the script is currently deployed based on a bunch of hacky Apache rules

Doesn't this mean the hacky Apache rules are now broken since the previous aur URL no longer works?

**Comment by stuartpb** [comment]
> Doesn't this mean the hacky Apache rules are now broken since the previous aur URL no longer works?

No, the hacky Apache rules are the mechanism it uses to serve the HTML version of the script to browsers and the text version to cURL. It has nothing to do with the format of AUR.

The script is broken as deployed because _the script itself_ has not been updated on the server (because I don't like FTPing the fragile Apache-dependent project formulation vs. git pushing a slug deployment of the Node-based meta.sh server).

**Comment by AndrewBelt** [comment]
Oh, I see. I misunderstood the context of that statement.

**Comment by stuartpb** [comment]
I should probably get the aur4 version of this uploaded some time soon, since I'm starting to see the same patch as #5 get written by others.
